docs/reference/asyncapi-rules.md

```diff
@@ -0,0 +1,311 @@
+# AsyncAPI Rules
+
+Spectral has a built-in "asyncapi" ruleset for the [AsyncAPI Specification](https://www.asyncapi.com/docs/specifications/2.0.0/).
+
+In your ruleset file you can add `extends: "spectral:asyncapi"` and you'll get all of the following rules applied.
+
+## AsyncAPI v2-only
+
+These rules will only apply to AsyncAPI v2 documents.
+
+### asyncapi2-channel-no-empty-parameter
+
+Channel parameter declarations cannot be empty, ex.`/given/{}` is invalid.
+
+**Recommended:** Yes
+
+### asyncapi2-channel-no-query-nor-fragment
+
+Query parameters and fragments shouldn't be used in channel names. Instead, use bindings to define them.
```

Comment:
Can we get some examples?

Reply:
@philsturgeon I'm afraid I won't be able to come up with a good/bad example that would make sense in an AsyncAPI context.

@fmvilas Could you please help me here? What's the known bad practices regarding query and fragments and how should they be converted to bindings?